dashboard/src/components/DebounceInput/DebounceInput.tsx

```diff
@@ -3,24 +3,26 @@ import { ChangeEvent, useEffect, useRef, useState } from 'react';
 import { useDebounce } from '@/hooks/useDebounce';
 import { Input, InputProps } from '@/components/ui/input';
 
-interface IDebounceInput extends Omit<InputProps, 'onChange' | 'value'> {
-  debouncedInterval: number;
+interface IDebounceInput extends Omit<InputProps, 'value'> {
   debouncedSideEffect: (value: React.ChangeEvent<HTMLInputElement>) => void;
   startingValue?: string;
 }
 
+const DEBOUNCE_INTERVAL = 666;
+
 const DebounceInput = ({
-  debouncedInterval,
   debouncedSideEffect,
   startingValue = '',
+  onChange,
```

Comment:
I think this is a leftover, you are not using in the TreeListingPage and it feels weird since we have the debouncedSideEffect